harmony_regridding_service/resample.py

```diff
@@ -72,21 +72,35 @@ def resample_n_dimensional_variables(
     processed = set()
 
     for var_name in variables:
-        processed.add(var_name)
         logger.debug(f'resampling {var_name}')
-        resampler = resampler_cache[horizontal_dims_for_variable(var_info, var_name)]
+        try:
+            resampler = resampler_cache[
+                horizontal_dims_for_variable(var_info, var_name)
+            ]
 
-        (s_var, t_var) = copy_var_with_attrs(source_ds, target_ds, var_name)
+            target_dimensions = get_preferred_ordered_dimension_names(
+                var_info, var_name
+            )
 
-        # We have to get the fill value off of the variable here because we
-        # only pass the numpy.ndarray into resample_variable_data and we need
-        # to know the fill value for the actual resampler.compute call.
-        fill_value = getattr(t_var, '_FillValue', None)
+            (s_var, t_var) = copy_var_with_attrs(
+                source_ds, target_ds, var_name, override_dimensions=target_dimensions
+            )
 
-        t_var[:] = resample_variable_data(
-            s_var[:], t_var[:], resampler, var_info, var_name, fill_value
-        )
-        logger.debug(f'Processed: {var_name}')
+            # We have to get the fill value off of the variable here because we
+            # only pass the numpy.ndarray into _resample_variable_data and we need
+            # to know the fill value for the actual resampler.compute call.
+            fill_value = getattr(t_var, '_FillValue', None)
+
+            source_variable = order_source_variable(s_var[:], var_info, var_name)
+
+            t_var[:] = resample_variable_data(
+                source_variable, t_var[:], resampler, var_info, var_name, fill_value
+            )
+            processed.add(var_name)
+            logger.debug(f'Processed: {var_name}')
+        except Exception as e:
+            logger.info(f'Failed to process: {var_name}')
+            logger.info(e)
```

Comment:
This is kind of bad for now, what happens is that a variable like the string utc var is processed all the way through to the resampling stage where it excepts because it can't be converted to a float64.  But now at least we just catch the exception and keep processing.  The bad part is that the output file will still have that variable because you can't delete it after it's been created.  I will handle this case with the Excluded Science Variables and think how to handle it in the future.